server/data/series.js

```diff
@@ -19,6 +19,17 @@ export function getSeriesCommissionedLength(seriesUrl: string): ?number {
 
 export const series = List([
   ({
+    url: 'body-squabbles',
+    name: 'Body Squabbles',
+    items: List([
+      ({
+        contentType: 'comic',
+        headline: 'Demodicid Navigation',
+        url: ''
+      }: ArticleStub)
+    ])
+  }: Series),
```

Comment:
No – it's [effitively infinite](https://github.com/wellcometrust/wellcomecollection.org/issues/663#issuecomment-296601431)